app/controllers/coronavirus_form/check_answers_controller.rb

```diff
@@ -25,6 +25,8 @@ def submit
         UnixTimestamp: Time.zone.now,
         FormResponse: session,
       )
+
+      send_confirmation_email if session.dig(:contact_details, :email).present?
```

Comment:
I deployed this in a second step for the other app, but we're more confident this time I don't see why shouldn't do it all in one step